desktop/src/com/limegroup/gnutella/gui/StatusLine.java

```diff
@@ -457,7 +480,7 @@ private void updateCenterPanel() {
 
     private void updateSeedingStatus() {
         boolean seedingStatus = SharingSettings.SEED_FINISHED_TORRENTS.getValue();
-        seedingStatusButton.setText("<html>"+(seedingStatus ? I18n.tr("<b>Seeding</b>") : I18n.tr("<b>Not Seeding</b>"))+"</html>");
+        seedingStatusButton.setText("<html>" + (seedingStatus ? I18n.tr("<b>Seeding</b>") : I18n.tr("<b>Not Seeding</b>")) + "</html>");
```

Comment:
let's use this opportunity to move the `<b>` tag out of the `I18n.tr` call, just leave the `Seeding` in.